internal/tasks/metrics.go

```diff
@@ -41,6 +41,14 @@ var (
 		Name: "keppel_retried_vulnerability_checks",
 		Help: "Counter for vulnerability checks that were retried due to transient errors in Clair.",
 	})
+	checkPendingVulnerabilitySuccessCounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "keppel_successful_pending_vulnerability_checks",
+		Help: "Counter for successful updates of the pending vulnerability status of a manifest.",
+	})
+	checkPendingVulnerabilityFailedCounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "keppel_failed_pending_vulnerability_checks",
+		Help: "Counter for failed updates of the pending vulnerability status of a manifest.",
+	})
```

Comment:
I don't like these metric names. It's not a "pending vulnerability check", it's a vulnerability check for an image in vulnerability status "Pending". I don't have a good idea for a succinct metric name right now; just want to register my dissatisfaction with these names for the moment.